test/spec/utils/analyzers/time_tracker_spec.rb

```diff
@@ -191,7 +220,7 @@ def previous_list
     end
 
     def previous_date
-      actions.last.date.to_date
+      actions.last.date.to_datetime.utc.to_date
```

Comment:
If we (or Trello) are in control of all these actions, they should all be UTC already.